pts/build.gradle

```diff
@@ -41,9 +37,9 @@ dependencies {
 
 jacocoTestReport {
     reports {
-        xml.enabled true
-        html.enabled true
-        csv.enabled false
-        html.destination file("${buildDir}/reports/jacoco/html")
+        xml.required.set(false)
+        html.required.set(true)
+        xml.required.set(false)
```

Comment:
same duplication

Reply:
Nice catch! I think it makes sense to move them all in the main `build.gradle` and remove the duplicate altogether.
Will make the change in the next commit.